### PR TITLE
Prac 5 feedback

### PR DESCRIPTION
**Pull request by Anastasia-Eremenko**
Completed prac 5 - pull request exercise
@ChristineYamin
@Quinn368

### CODE REVIEW

**Comment by Quinn368** [comment]
Great program!!!

File: CP1404_an/Practical_05/state_names.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+for code, name in CODE_TO_NAME.items():
+    print(f"{code:<3} is {name}")
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    try:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    except KeyError:
+        print("Invalid short state")
+    state_code = input("Enter short state: ").upper()
```
Comment (ChristineYamin) [inline CP1404_an/Practical_05/state_names.py:18]: Clear code 👍

File: CP1404_an/Practical_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"""
+CP1404 Practical
+Colour names and codes in a dictionary
+"""
+
+colors_dict = {"absolute zero": "#0048ba", "acid green": "#b0bf1a", "alice blue": "#f0f8ff",
+               "alizarin crimson": "#e32636", "amaranth": "#e52b50", "amber": "#ffbf00",
+               "amethyst": "#9966cc", "antique white": "#faebd7", "antique white 1": "#ffefdb",
```
Comment (ChristineYamin) [inline CP1404_an/Practical_05/hex_colours.py:8]: According to instruction, you have to write only 10 colour and code.

File: CP1404_an/Practical_05/word_occurrences.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+def find_longest_word(words):
+    """Finds the length of the longest word"""
+    longest_word = 0
+    for word in words:
+        if len(word) > longest_word:
+            longest_word = len(word)
+    return int(longest_word)
+
+
+main()
```
Comment (ChristineYamin) [inline CP1404_an/Practical_05/word_occurrences.py:42]: Clear and nice code

**Comment by ChristineYamin** [comment]
All of your codes are unique and they are nice programs.

**Pull request merged**